src/platform.rs

```diff
@@ -27,28 +27,12 @@ where
     /// ```no_run
     ///use libcnb::platform::Platform;
     ///use libcnb::platform::GenericPlatform;
+    /// use libcnb::generic::GenericPlatform;
```

Comment:
It seems like the `libcnb::platform` lines can now be removed? (The indentation is also inconsistent)

Reply:
Yeah, this has been removed. :)